pkg/cri/opts/spec_linux_opts.go

```diff
@@ -103,6 +103,13 @@ func WithMounts(osi osinterface.OS, config *runtime.ContainerConfig, extra []*ru
 				dst = mount.GetContainerPath()
 				src = mount.GetHostPath()
 			)
+
+			// Destination must be an absolute path.
+			// If it isn't, let's make it relative to "/".
```

Comment:
Is there any reason to add this? I think this is behavior change. Is it related to your test case?

Reply:
It shouldn't be behavior change in the sense that a relative path is made relative to `/` today.

This is needed because runc _right now_ throws an error if using a relative destination path, it should be an abs path. As I say in my other message, we can maybe postpone that upcoming change in runc

Reply:
We need to change this for linux, so changing only in `pkg/cri/opts/spec_linux_opts.go` seems to be enough for linux! Removed this and all the other test changes that were brought by this line :)